### DISCUSSION TITLE
"no supported display found"?

### DISCUSSION THREAD

**Discussion by captamerican** [MonitorControl/MonitorControl#376] [Q&A] [answered]
Downloaded the latest version of the app but says "no supported display found"
Is there a way around this?
LG 32UD99 4K 32 inch display connected to M1 Mac Mini

**Comment by scurrie80** [comment]
Yep, it's a known issue (#323) and appears to be related to a bug in Big Sur.  Bottom line the issue is blocked until Apple fixes their bug first.

**Comment by waydabber** [comment, marked as answer]
Hi, v3.0.0 is released with full M1 support. Be aware however that the M1 mini's HDMI port is not supported (just like the 2018 Intel mini), you need to use the USB-C port.
